jsonrpc-specs/src/content/results.json

```diff
@@ -0,0 +1,208 @@
+    "description": "Returns CONTENT message with a content or, in case the recipient does not have the data, a list of ENR records of nodes that are closer than the recipient is to the requested content.",
+    "schema": {
+      "title": "CONTENT message",
+      "type": "object",
+      "properties": {
+        "connectionId": {
+          "description": "uTP connection ID",
+          "schema": {
+            "type": "number"
+          }
+        },
+        "content": {
+          "description": "Requested content",
+          "$ref": "#/components/schemas/hexString"
+        },
+        "enrs": {
+          "description": "list of ENR records of nodes that are closer than the recipient is to the requested content",
+          "type": "array",
+          "items": {
+            "$ref": "#/components/schemas/Enr"
+          }
+        }
+      }
+    }
+  },
+  "FindNodeResult": {
+    "name": "findNodeResult",
+    "description": "Returns nodes in a given distance",
+    "schema": {
+      "title": "NODES message",
+      "type": "array",
+      "items": {
+        "$ref": "#/components/schemas/Enr"
+      }
+    }
+  },
+  "GetEnrResult": {
+    "name": "getEnrResult",
+    "description": "Returns latest ENR associated with the given node ID.",
+    "schema": {
+      "title": "Ethereum node record",
+      "$ref": "#/components/schemas/Enr"
+    }
+  },
+  "LookupEnrResult": {
+    "name": "lookupEnrResult",
+    "description": "Returns ENR associated with the given node ID",
+    "schema": {
+      "title": "Ethereum node record",
+      "$ref": "#/components/schemas/Enr"
+    }
+  },
+  "OfferResult": {
+    "name": "offerResult",
+    "description": "Returns ACCEPT message",
+    "schema": {
+      "title": "ACCEPT message",
+      "type": "object",
+      "properties": {
+        "connectionId": {
+          "description": "Connection ID to set up a uTP stream to transmit the requested data",
+          "$ref": "#/components/schemas/bytes2"
+        },
+        "contentKeys": {
+          "description": "Signals which content keys are desired",
+          "$ref": "#/components/schemas/hexString"
+        }
+      }
+    }
+  },
+  "PingResult": {
+    "name": "pingResult",
+    "description": "Returns PONG response",
+    "schema": {
+      "title": "PONG message",
+      "type": "object",
+      "required": [
+        "enrSeq",
+        "recipientIP",
+        "recipientPort"
+      ],
+      "properties": {
+        "enrSeq": {
+          "description": "ENR sequence number of sender",
+          "type": "number"
+        },
+        "recipientIP": {
+          "description": "IP address of the intended recipient",
+          "$ref": "#/components/schemas/ipAddr"
+        },
+        "recipientPort": {
+          "description": "recipient UDP port",
+          "$ref": "#/components/schemas/udpPort"
+        },
+        "payload": {
+          "description": "Custom payload data",
+          "$ref": "#/components/schemas/hexString"
```

Comment:
I guess here we want to rename this field to `dataRadius` and return a `data radius` value instead of a payload hex string (at least for portalState and portalHistory rpc)?